cabot/cabotapp/models.py

```diff
@@ -141,6 +144,17 @@ class Meta:
         help_text='Gist, Hackpad or Refheap js embed with recovery instructions e.g. '
                   'https://you.hackpad.com/some_document.js'
     )
+    hipchat_instance = models.ForeignKey(
```

Comment:
Should we have it in CheckGroupMixin or in Service? (CheckGroupMixin is shared by Instance - which we don't use).